frontend/app/_api/api.ts

```diff
@@ -59,7 +65,7 @@ api.interceptors.response.use(
     if (res.data?.token) {
       localStorage.setItem('access_token', res.data?.token)
     }
-    if (res.headers['authorization'].includes('Bearer')) {
+    if (res.headers['authorization']?.includes('Bearer')) {
```

Comment:
Is this what you were referring to when mentioning that you want a django middleware to fetch the new access token and return it in the response headers?

Why can't api.ts just make a call (new axios instance or even just fetch) to /jwt/refresh/ when the access token stops working?